browser/js/dashboard/newDashboard/newDash.controller.js

```diff
@@ -0,0 +1,15 @@
+app.controller('newDashboardCtrl', function (userFactory,$state,$uibModalInstance, $scope) {
+	
+	// $scope.form = {
+	// 	name: dashbaord.name, 
+	// 	description: dashboard.description
+	// }
```

Comment:
Best practice is to avoid pushing commented-out code to GitHub. If the code was in an earlier commit, you can always go back to that commit to find it.